packages/fuels-accounts/src/provider.rs

```diff
@@ -166,12 +178,18 @@ impl Provider {
         tx: T,
     ) -> Result<TxStatus> {
         #[cfg(feature = "coin-cache")]
-        self.check_inputs_already_in_cache(&tx.used_coins(self.base_asset_id()))
+        let base_asset_id = {
+            let consensus_parameters = self.consensus_parameters().await?;
+            *consensus_parameters.base_asset_id()
+        };
```

Comment:
```suggestion
        let base_asset_id = *self.consensus_parameters().await?.base_asset_id();
```

Reply:
Ty. Gonna do these in a separate PR just so I  don't loose the precious approves.